src/widget.cpp

```diff
@@ -1493,7 +1493,7 @@ std::string widget::layout( const avatar &ava, unsigned int max_width, int label
                 const std::string txt = cur_child.layout( ava, skip_pad_this ? 0 : cur_width,
                                         label_width, skip_pad_this );
                 // Store the resulting text for this column
-                cols.emplace_back( foldstring( txt, skip_pad_this ? 0 : cur_width ) );
+                cols.emplace_back( string_split( txt, '\n' ) );
```

Comment:
It looks like this change is causing some test failures.

Reply:
I changed the tests to fit the fix. The failure is in a layout that does some strange things. This is worth taking a look at before merging.